docs/guides/upgrades/v0.38-to-v1.0.md

```diff
@@ -0,0 +1,598 @@
+---
+order: 4
+title: CometBFT v0.38.x to v1.0
+---
+
+# Upgrading from CometBFT v0.38.x to v1.0
+
+This guide outlines the key updates and adjustments needed when upgrading from CometBFT
+v0.38.x to v1.0. It highlights significant changes and provides references to additional
+documentation for a smoother transition.
+
+The introduction of CometBFT `v1.0` brings numerous new features. This guide outlines the
+most significant additions and changes.
+
+For additional information about changes please refer to the [CHANGELOG.md](https://github.com/cometbft/cometbft/blob/main/CHANGELOG.md)
+and [UPGRADING.md](https://github.com/cometbft/cometbft/blob/main/UPGRADING.md) documents.
+
+> **NOTE**: It is essential to emphasize that this is a [major version bump](https://github.com/cometbft/cometbft/blob/main/README.md#versioning)
+> upgrade (`v0.38.x` -> `v1.0`). Therefore, a **coordinated upgrade** is necessary. A mixed network with `v0.38.x` and `v1.0` nodes is **not supported**.
+
+## Proposer-Based Timestamps (PBTS)
+
+CometBFT `v1.0` contains a new algorithm for generating and verifying block timestamps
+called Proposer-Based Timestamps (`PBTS`).
+The existing algorithm used in CometBFT releases prior to `v1.0`, called [BFT-Time][bft-time] is kept for backwards compatibility.
+
+Upgrading to `v1.0` **does not automatically switch the chain from BFT-Time
+to PBTS**; rather a `ConsensusParam` called `PbtsEnableHeight` can be set to a future
+height to transition from BFT-Time to PBTS.
+
+This flexible mechanism allows chains disentangle the upgrade to `v1.0` from the transition
+in the algorithm used for block times.
+
+For further information, please check the [PBTS specification][pbts-spec].
+
+## ABCI Mutex
+
+CometBFT's existing `ABCI` local client is prevented from making
```

Comment:
I think it is not clear for everyone what an ABCI client is.

In any case, I think we should mention that this only applies to the case where Comet and the ABCI application run in the same process.